src/views/start.njk

```diff
@@ -54,18 +54,40 @@
         If you need assistance with this application, contact the NatureScot Licensing Team.
```

Comment:
katherine would like this text to be - 'If you need assistance with this service, contact the NatureScot Licensing Team'